src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(String color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public void print() {
+        super.print();
+        System.out.println("circle,"
+                + " area: " + getArea()
+                + " sq.units,"
+                + " radius: "
+                + radius + " sq.units, "
+                + "color: " + getColor());
```

Comment:
Creating a figure, don't pass expressions in the constructor. It's better to create separate variables for the area and color and pass them on for better code readability. Also, all magic numbers in your code should be constants. The 'sq.units' seems to be used as a unit of measurement and should be extracted as a constant.